stock_warehouse_calendar_orderpoint/models/res_company.py

```diff
@@ -10,11 +10,15 @@ class ResCompany(models.Model):
     orderpoint_calendar_id = fields.Many2one(
         comodel_name="resource.calendar",
         string="Reordering Calendar",
-        help="Calendar used to compute the lead date of reordering rules",
+        help="Calendar used to compute the lead date of reordering rules\n"
+        "This value will be used as default value for new warehouses linked to this"
+        " company",
     )
     orderpoint_on_workday = fields.Boolean(
         string="Schedule the lead date on workday only",
-        help="Postpone the lead date to the first available workday",
+        help="Postpone the lead date to the first available workday\n"
+        "This value will be used as default value for new warehouses linked to this"
+        " company",
     )
     orderpoint_on_workday_policy = fields.Selection(
```

Comment:
do we still need `orderpoint_on_workday` ? `orderpoint_on_workday == False` is the same as `orderpoint_on_workday_policy == False`, and `orderpoint_on_workday == True` is the same as `orderpoint_on_workday_policy != False`, right ?

Reply:
Uhm, makes sense, it would make the configuration quicker.
I'll update the PR with this suggestion.

Reply:
do not forget to write a migration script, as we do not want to lose the current configuration ^^

Reply:
The module doesn't exist in the repo yet, so we may need to do it on a project-by-project base (for those projects that already use it).

Reply:
Yeah, but it's already installed on our customer instance, right ?
Better make the migration script public.
You have to do it anyways.